robot_smach_states/src/robot_smach_states/human_interaction/find_person_in_room.py

```diff
@@ -14,21 +14,35 @@
 import robot_smach_states as states
 import robot_smach_states.util.designators as ds
 from robot_skills.util import kdl_conversions
-
+# class CheckIfPersonInRoom(smach.State):
+#     def __init__(self, robot, room):
+#         """
+#
+#         :param robot: robot api object
+#         :param room: room where person should be found
+#         """
+#         smach.State.__init__(self, outcomes=['true', 'false'])
+#         self._robot = robot
+#         self._room = room
+#
+#     def execute(self, userdata=None):
 
 class FindPerson(smach.State):
     """ Smach state to find a person. The robot looks around and tries to recognize all faces in view.
 
         """
     # ToDo: robot only mentions that it has found the person. Doesn't do anything else...
 
-    def __init__(self, robot, person_label='operator', lost_timeout=60, look_distance=2.0, probability_threshold=1.5):
+    def __init__(self, robot, person_label='operator', lost_timeout=60, look_distance=1.0, probability_threshold=1.5,
```

Comment:
Weird variable name, lost_timeout, for when you are looking/searching

Reply:
Proposed and implemented new name is search_timeout